app/models/ScratchResponse.scala

```diff
@@ -18,9 +18,9 @@ package models
 
 import play.api.libs.json.{Json, OFormat}
 
-case class SaveScratchSubmissionResponse(id: String)
+case class ScratchResponse(id: String)
```

Comment:
If we were to change ApprovalResponse to SaveApprovalResponse it would nice to rename this class also. However, as we are probably implementing save only there isn't the need to distinguish between responses as we have with approval.

Reply:
As above